ee/orchestration/internal/triggers/activity.go

```diff
@@ -74,35 +76,42 @@ func (a Activities) ProcessTrigger(ctx context.Context, trigger Trigger, request
 
 	span := trace.SpanFromContext(ctx)
 	var (
-		evaluated map[string]string
-		err       error
+		evaluated    map[string]string
```

Comment:
There, we return a specific error on variables evaluation failures. This error is non retryable.